Sources/SafeDITool/SafeDITool.swift

```diff
@@ -52,6 +52,13 @@ struct SafeDITool: AsyncParsableCommand, Sendable {
 			throw ValidationError("Must provide 'swift-sources-file-path', '--include', or '--include-file-path'.")
 		}
 
+		async let existingGeneratedCode: String? = Task.detached {
+			if let dependencyTreeOutput {
+				try? String(contentsOf: dependencyTreeOutput.asFileURL, encoding: .utf8)
+			} else {
+				nil
+			}
+		}.value
 		let (dependentModuleInfo, module) = try await (
```

Comment:
```suggestion
		}.value

		let (dependentModuleInfo, module) = try await (
```
nit; Extra space after `existingGeneratedCode` computation.